ui/README.md

```diff
@@ -0,0 +1,14 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Local Development using VSCode Chrome Debugger
+
+Run:
+
+### `npm start`
+
+Runs the app in the development mode on localhost:3000<br>
```

Comment:
I think these steps should be converted to an ordered list so it is easier to follow.